examples/mixture_models/dirichlet_mixture_of_multinomials.ipynb

```diff
@@ -0,0 +1,1819 @@
+{
```

Comment:
<p>bit confused about what his <code>None</code> is, in the second line of this cell</p><p> </p><p>EDIT: just saw Alex's comments on this, agree that <code>;</code>  would be better</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Seconded</p>

Reply:
<p>Yeah, that seems to be most folks' preference... :-/</p><p> </p><p>I'll make that change for consistency with the other examples.</p><p> </p><p>(FWIW: My arguments for <code>None</code> is that  the <code>;</code>  at the end of a line is (A) hidden and (B) can be confusing for python beginners who don't really see semicolons anywhere else in the language. I'd also guess that (C) with enough copy-pasting semicolons will start polluting other lines.)</p>